msbuild/Xamarin.iOS.Tasks.Core/Tasks/LocalizationStrings.cs

```diff
@@ -0,0 +1,38 @@
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Xamarin.MacDev.Tasks;
+using Xamarin.Localization.MSBuild;
+using System.ComponentModel;
+using System.Reflection;
+
+namespace Xamarin.iOS.Tasks
+{
+	public abstract class LocalizationStrings : XamarinTask
+	{
+		public string MSBErrorCode { get; set; }
+
+		public override bool Execute ()
```

Comment:
Is this a task we only use for tests?

Reply:
If I understand your question correctly, yes this task would just be for tests.

Reply:
This task is in the main Xamarin.iOS.Tasks.Core assembly, so we'd be shipping it to customers.

I'm not sure this is the best approach. Maybe @rolfbjarne has thoughts?

Reply:
Oh okay yeah sure that makes sense!

Reply:
I think a better idea would be to reference the Xamarin.Localization.MSBuild project from the test project, and just call `MSBStrings.E0007` directly, instead of going through a task.

Also you might want to wait a bit, since there will be changes in this area (in particular #9860 will change the msbuild tests quite a bit, and more changes are coming soon).

Reply:
Okay will do! Also with calling 'MSBStrings.E0007' directly, I get an error saying it cannot find the namespace Xamarin.Localization.MSBuild which is where MSBStrings resides. Any idea why?

Reply:
> I get an error saying it cannot find the namespace Xamarin.Localization.MSBuild

I'm guessing that you need to add a reference to the Xamarin.Localization.MSBuild project in the test project